sdk/keyvault/Microsoft.Azure.KeyVault.Core/src/Microsoft.Azure.KeyVault.Core.csproj

```diff
@@ -1,7 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Description>Microsoft Azure Key Vault Core Class Library</Description>
+    <Description>Microsoft Azure Key Vault Core Class Library
+
```

Comment:
Should consider sharing these common properties in the props file.

Reply:
1. Why complicate things with a partial property (i.e. the package descriptions are all different sans this blurb)?
2. This is only for track 1 packages.

Reply:
OK from a quick glance I thought they were all the same.